src/txn/txn.c

```diff
@@ -2794,7 +2799,7 @@ __wt_verbose_dump_txn_one(
         ", durable_timestamp: %s"
         ", first_commit_timestamp: %s"
         ", prepare_timestamp: %s"
-        ", pinned_durable_timestamp: %s"
+        ", prepared id: %" PRIu64 ", pinned_durable_timestamp: %s"
```

Comment:
Can you move the `pinned_durable_timestamp` to the next line?

Reply:
this line is automatically appended to the end of `prepared id` by `s_clang_format`. It also doesnt make any difference in the output string so i dont think it's neccessary to put it to the next line